app/screens/select_server/index.js

```diff
@@ -21,9 +21,10 @@ function mapStateToProps(state) {
     const {currentVersion, latestVersion, minVersion} = getClientUpgrade(state);
 
     return {
-        ...state.views.selectServer,
+        serverUrl: getCurrentUrl(state),
```

Comment:
why are we using the entities general url? this should use the one in the view state as it was... can you clarify this change?

Reply:
this should actually be reverted back to it's previous state